exercises/yacht/src/test/java/YachtTest.java

```diff
@@ -5,192 +5,194 @@
 
 public class YachtTest {
 
+    private Yacht yacht;
```

Comment:
Very minor thing but since this is constructed in each test it doesn't have to be a private variable. Making it local to each test will make it more clear that it's different in each test and not shared between them 🙂

Reply:
Of course. As I said I took [ScrabbleScoreTest.java](https://github.com/exercism/java/blob/master/exercises/scrabble-score/src/test/java/ScrabbleScoreTest.java) as an example to try to be consistent. Maybe the private field should also be removed there, then?

Reply:
I see, thanks for pointing that out! I'll open an issue for it 🙂